hathor/p2p/resources/status.py

```diff
@@ -99,6 +99,11 @@ def render_GET(self, request):
         raw_best_blockchain = self.manager.tx_storage.get_n_height_tips(self._settings.DEFAULT_BEST_BLOCKCHAIN_BLOCKS)
         best_blockchain = to_serializable_best_blockchain(raw_best_blockchain)
 
+        whitelist = self.manager.connections.peers_whitelist
+        if whitelist is not None:
+            peer_ids = whitelist._current
```

Comment:
You are accessing an internal attribute. It's always better to create a `get_all_peers()` method and simply call it from here.